scholia/app/templates/chemical.html

```diff
@@ -50,7 +50,7 @@ <h2 id="Identifiers" id="identifiers">Identifiers</h2>
 
 <table class="table table-hover" id="identifiers-table"></table>
 
-<h2 id="Related" id="related">Related Compounds</h2>
+<h2 id="Related" id="related">Compounds with same connectivity</h2>
```

Comment:
Should the id  be changed accordingly to avoid similar confusion later?

Reply:
There is a duplication here and some other places. I propose we fix that in a different patch: we need to check all files.

Reply:
I am not a big fan overloading patches.

Reply:
Just to not forget about it, I filed https://github.com/WDscholia/scholia/issues/2226

Reply:
OK.